routes/web.php

```diff
@@ -81,6 +81,7 @@
 
 // Articles
 Route::prefix('articles')->namespace('Articles')->group(function () {
+    Route::get('me', 'MyArticles')->name('user.articles');
```

Comment:
Maybe `/me` isn't a good choice here as it implies yourself as a person. `/created`?